local.properties

```diff
@@ -1,10 +1,8 @@
-## This file is automatically generated by Android Studio.
-# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
-#
-# This file should *NOT* be checked into Version Control Systems,
+## This file must *NOT* be checked into Version Control Systems,
 # as it contains information specific to your local configuration.
```

Comment:
crap. I missed this in the gitignore. lets chat about nuking this.

Reply:
Added a commit that nuked it.